src/Views/UserSettingsView.vala

```diff
@@ -345,13 +407,22 @@ namespace SwitchboardPlugUserAccounts.Widgets {
                 user_type_dropdown.sensitive = false;
                 password_button.sensitive = false;
                 autologin_switch.sensitive = false;
+                if (fp_utils != null) {
+                    remove_fp_button.sensitive = false;
+                    fingerprint_button.sensitive = false;
+                }
 
                 autologin_label.secondary_text = NO_PERMISSION_STRING;
                 user_type_label.secondary_text = NO_PERMISSION_STRING;
             }
 
             lang_label.secondary_text = null;
 
+            if (fp_utils != null) {
+                remove_fp_button.sensitive = current_user && allowed && fp_utils.is_enrolled ();
+                fingerprint_button.sensitive = current_user && allowed;
```

Comment:
activating these buttons should include a step to get permission so we shouldn't need to check for `allowed` here